src/main/java/Duke.java

```diff
@@ -6,5 +8,82 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String dashes = "____________________________________________________________";
+
+        String intro = "Hello! I'm Duke\nWhat can I do for you?";
+        System.out.println(intro);
+        System.out.println(dashes);
+        Task[] List = new Task[100];
+        String[] List2 = new String[100];
+        int Count = 0;
+        Scanner sc = new Scanner(System.in);
+        while (sc.hasNextLine()) {
+            String line;
+
+            line = sc.nextLine();
+            try {
+                if (line.equals("bye")) {
+                    break;
+                }
+                if (line.equals("list")) {
+                    System.out.println(dashes);
+                    System.out.println("Here are the tasks in your list:");
+
+                    for (int i = 0; i < Count; i++) {
+                        System.out.println((i + 1) + "." + List[i].getDescription());
+                    }
+                    System.out.println(dashes);
+                } else if (line.split(" ")[0].equals("done")) {
+                    int c = Integer.parseInt(line.split(" ")[1]);
+                    List[c - 1].markAsDone();
+                    System.out.println("Nice! I've marked this task as done: ");
+                    System.out.println((c) + "." + List[c - 1].getDescription());
+                } else {
+                    System.out.println(dashes);
+                    String desc;
+                    switch (line.split(" ")[0]) {
```

Comment:
Could follow the style for switch statements. Please refer to the coding standard